### PR TITLE
Adds Youtube-style Scrubbing preview

### PR DESCRIPTION
**Pull request by dsernst**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[x] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

This PR makes it is easier to find the right spot when you're quickly skipping through a video.

It adds a visual preview as your mouse cursor hovers over the scrubbing bar.

Here's an example (using a Charlie Chaplin video from [archive.org](http://archive.org)):

### Old (no preview): 

Kapture 2020-05-01 at 22 27 41

### New (with preview):

Kapture 2020-05-01 at 22 25 34

**Which issue (if any) does this pull request address?**

https://github.com/webtorrent/webtorrent-desktop/issues/1811

👍 Tested and works locally on my machine, running macOS 10.15.

### CODE REVIEW

**Comment by dsernst** [comment]
This preview:
- [x] only appears for video files, not audio
- [x] maintains a constant height (70px), but auto adjusts its width to maintain video aspect ratio
- [x] avoids falling off the left or right edges

**Comment by dsernst** [comment]
Hey friends—  this PR quickly got a lot of `heart emojis` (🙏 thank you @hicom150 @alxhotel @Krusty27 @MomenNano @subins2000 @jaytohe) but now it's been stuck in review limbo for the last few months. 

Lmk if there's something I can do. Happy to ship a fix if necessary. Or maybe chime in just to say if you want this or have concerns or something? 🙂 Something to move the ball forward.

I'd like to use this and ship it to my friends that use WD.
Would rather not have to start maintaining a separate fork if not necessary 😅

**Comment by subins2000** [comment]
This would certainly be a great addition to WebTorrent player. The maintainers of this repo have all been pretty busy I guess, many open PRs and issues. Hopefully it'll be reviewed soon.

File: src/renderer/pages/player-page.js [hunk 1]
```diff
@@ -722,6 +726,18 @@ function renderPlayerControls (state) {
...
+  // Handles a scrub hover (preview another position in the video)
+  function handleScrubPreview (e) {
+    // Only show for videos
+    if (!e.clientX || state.playing.type !== 'video') return
+    dispatch('mediaMouseMoved')
+    dispatch('preview', e.clientX)
+  }
+
+  function clearPreview (e) {
+    dispatch('clearPreview')
```
Comment (hicom150) [inline src/renderer/pages/player-page.js:738]: I would add the same check that in `handleScrubPreview` 😉
```
if (state.playing.type !== 'video') return
```

Comment (dsernst) [reply src/renderer/pages/player-page.js:739]: Done. Thank you 🙏

**Pull request merged**